red-hat-data-services/odh-manifests
# Sync dashboard manifest with upstream files

## Description

This PR is targeted to sync downstream with upstream, it will need a follow up with the kfdefs for the correct overlay.
The main changes are:

- Remove old files
- Add a new rhods overlay with the changes for rhods

## Testing instructions

You can test the following kfdef:

```yaml
apiVersion: kfdef.apps.kubeflow.org/v1
kind: KfDef
metadata:
  name: rhods-dashboard
spec:
  applications:
  - kustomizeConfig:
      parameters:
      - name: odh-dashboard-image
        value: ${RELATED_IMAGE_ODH_DASHBOARD_IMAGE} //change this with the targeted image
      overlays:
        - rhods
      repoRef:
        name: manifests
        path: odh-dashboard
    name: odh-dashboard
  repos:
  - name: manifests
    uri: https://github.com/lucferbux/odh-manifests/tarball/sync-dashboardconfig-upstream
  version: v1.0.0
```

To check if the operator can add the overlay, the other testing method is:

1. Go to the `master` branch and run `kustomize build odh-dashboard/overlays/authentication`
2. Copy the text to a git differ
3. No switch to this branch
4. Run `kustomize build odh-dashboard/overlays/rhods`
5. Check the diff, there should be several additions and removals, most of them new changes that weren't tracked downstream, but all the names should be `rhods-dashboard` instead of `odh-dashboard`

Or install the live build:

`quay.io/lferrnan/rhods-operator-live-catalog:1.31.0-sync`

## Merge criteria:



- [x] JIRA link(s): https://issues.redhat.com/browse/RHODS-10169, https://issues.redhat.com/browse/RHODS-10170
